### PR TITLE
fix uprobe address on short name

### PR DESCRIPTION
**Pull request by fbs**
Fixes: #972

### CODE REVIEW

File: tests/ast.cpp [hunk 1]
```diff
@@ -51,11 +51,15 @@ TEST(ast, probe_name_uprobe)
   AttachPointList attach_points3 = { &ap1, &ap2, &ap3 };
   Probe uprobe3(&attach_points3, nullptr, nullptr);
   EXPECT_EQ(uprobe3.name(), "uprobe:/bin/sh:readline,uprobe:/bin/sh:somefunc,uprobe:/bin/sh:1000");
-
```
Comment (ajor) [inline tests/ast.cpp]: Could you leave this spacing line in please? I feel it makes the tests a lot more readable

Comment (fbs) [reply tests/ast.cpp]: Oops, sloppy.

**Comment by fbs** [comment]
Fixed @ajor

**Comment by ajor** [comment]
Thanks!

**Pull request merged**